proms-resources/frontend/src/main/resources/SLING-INF/content/Extensions/Views/ToUConfiguration.json

```
@@ -0,0 +1,7 @@
+{
+  "jcr:primaryType": "cards:Extension",
+  "cards:extensionPointId": "cards/coreUI/view",
+  "cards:extensionName": "Terms of use configuration",
```

Change to "Patient Portal Terms of Use" per updated specs.
```suggestion
  "cards:extensionName": "Patient Portal Terms of Use",
```